AutoRest/Generators/NodeJS/NodeJS/Templates/MethodGroupIndexTemplateTS.cshtml

```diff
@@ -0,0 +1,19 @@
+﻿@using Microsoft.Rest.Generator.NodeJS
+@using Microsoft.Rest.Generator.NodeJS.Templates
+@using Microsoft.Rest.Generator.Utilities
+@using System.Linq
+@inherits Microsoft.Rest.Generator.Template<Microsoft.Rest.Generator.NodeJS.ServiceClientTemplateModel>
+/*
+@Header(" * ")
+*/
+@EmptyLine
+/// <reference path="../typings/tsd.d.ts" />
+import { RequestOptions, WebResource } from "../node_modules/ms-rest/lib";
+@if (Model.ModelTypes.Any()) {
+@:import * as models from "../models";
```

Comment:
That code is output in the "operations" subdirectory, so "../models" is needed to point to the sibling "models" directory.   So it should be OK.